src/mcts/utils.h

```diff
@@ -159,6 +190,17 @@ size_t sampleDiscreteProbability(size_t nElements,
   if (nElements == 0) {
     throw std::runtime_error("sampleDiscreteProbability was passed 0 elements");
   }
+  // First things first, we print out the probability distribution
+  float happysum = 0.0f;
+  for (size_t i = 0; i != nElements; ++i) {
+    happysum += getValue(i);
```

Comment:
Why is this `happysum` variable created if it's never used later?

Reply:
Good one, I forgot to comment this out. It's for printing out the probability distribution.

Reply:
I generally recommend deleting instead of commenting out unless you think you're going to have to debug this thing here again.

This kind of thing should be a test.

Reply:
I think there's a decent enough chance we use it again. Indeed, we'd like to be logging this information when we do the run, so the code will probably be re-used for that, and then it can be deleted from here.

Reply:
still maybe comment it out?